foam_controlwrapper/io_utils.py

```diff
@@ -0,0 +1,34 @@
+""" Utility functions for file IO
+
+"""
+import simphonyfoaminterface as foamface
+
+from .foam_mesh import FoamMesh
+
+
+def read_foammesh(name, path):
```

Comment:
No tests exist for this function.  Maybe an issue could be added to deal with this after this PR